Extension/AbstractTableBundleExtension.php

```diff
@@ -4,8 +4,6 @@
 
 abstract class AbstractTableBundleExtension implements TableBundleExtensionInterface
```

Comment:
Table bundle here? That's a different component? It appears some of the dependencies are messed up or there is a naming issue. Can you investigate and then we can discuss?